modules/op-configuring-pipelines-as-code-logging.adoc

```diff
@@ -0,0 +1,151 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-pipelines-as-code-logging_{context}"]
+= Configuring {pac} logging 
+
+You can configure {pac} logging by editing the `pac-config-logging` config map in the `TektonConfig` custom resource (CR).
+
+.Prerequisites
+
+* {pac} is installed on your cluster.
+
+.Procedure
+
+. In the *Administrator* perspective of the web console, go to *Administration* → *CustomResourceDefinitions*.
```

Comment:
Just with consistency on web console docs
```suggestion
. In the *Administrator* perspective of the web console, navigate to *Administration* → *CustomResourceDefinitions*.
```

Reply:
based on the IBM style guide: 
**navigate** Use only when a more direct verb, such as "click", "select", "browse", or "go to", is not applicable. Use only to provide direction to a location (for example, "Navigate to the directory that contains the files that you want to edit").

So I will keep "go" instead.

Reply:
If you choose to use "go" that is up to you like you mentioned the style guide does say you can use go; however, with the guideline "Use only to provide direction to a location" this would be a direction to a location with the final location being *CustomResourceDefinitions*. As I said, it is up to you but wanted to explain my reasoning further.